stores/metadata.go

```diff
@@ -1199,11 +1239,30 @@ func (s *SQLStore) CopyObject(ctx context.Context, srcBucket, dstBucket, srcPath
 		dstObj.ObjectID = dstPath     // set dst path
 		dstObj.DBBucketID = bucket.ID // set dst bucket id
 		dstObj.Slabs = srcSlices      // set slices
-		if err := tx.Create(&dstObj).Error; err != nil {
+		tx = tx.Create(&dstObj)
+		if err := tx.Error; err != nil {
```

Comment:
Is there a reason for this? You don't seem to use the `tx` after this point anyway?

Reply:
debug leftover